lib/ruby_home/config/characteristics.yml

```diff
@@ -419,6 +460,26 @@
       '1': Idle
       '2': Humidifying
       '3': Dehumidifying
+- :name: :current_media_state
+  :description: Current Media State
+  :uuid: 000000E0-0000-1000-8000-0026BB765291
+  :format: uint8
+  :unit: nil
+  :permissions:
+  - securedRead
+  :properties:
+  - read
+  - cnotify
+  :constraints:
+    MaximumValue: 5
+    MinimumValue: 0
+    ValidValues:
+      '0': Play
+      '1': Pause
+      '2': Stop
+      '3':
```

Comment:
I wasn't sure what to do with `ValidValues` that didn't have a mapping, so I just left them blank. This came from here: https://github.com/homebridge/HAP-NodeJS/blob/186ce56fab55b5868142e27b59af6c5ab53038d6/src/lib/gen/HomeKit-TV.ts#L126-L131